static/public.css

```diff
@@ -8,6 +8,37 @@
     --sc-blue-1: #CCE8f7;
     --sc-blue-2: #C6E1Ef;
 }
+@font-face {
```

Comment:
We do not want to load resources from external references for specific reasons so I will suggest the change of font is dropped and we could leave only the link decoration change and and the filter label text style